acceptance/bundle/validate/include_locations/databricks.yml

```diff
@@ -0,0 +1,10 @@
+bundle:
+  name: include_locations
+
+include:
+  - job.yml
```

Comment:
nit: would be interesting to put one of these in a directory as well to go through path rewriting here.